src/pages/index.js

```diff
@@ -67,7 +69,13 @@ const HomePage = ({ data }) => {
         isIconClickable
         alignIcon={SearchInput.ICON_ALIGNMENT.RIGHT}
         onChange={(e) => setSearchTerm(e.target.value)}
-        onSubmit={() => navigate(`?q=${searchTerm || ''}`)}
+        onSubmit={() => {
+          tessen.track({
+            eventName: 'defaultViewSearchTriggered',
+            category: 'HomepageSearch',
```

Comment:
I think these got flipped. The other search events look like this:
```suggestion
            eventName: 'homepageSearchEntered',
            category: 'DefautViewSearch',
```

Reply:
whoops, i decided halfway through that i liked the category to describe if it was default, persona, or sidebar. but the category could be the same for every one of them. maybe i'll ask austin if there's a preference here

Reply:
actually i think i'm just gonna make that change and have it all be one search category with varied events so we can narrow a select down by category and then facet on event